server/helpers/ffmpeg-utils.ts

```diff
@@ -116,28 +116,27 @@ type TranscodeOptions = {
 
 function transcode (options: TranscodeOptions) {
   return new Promise<void>(async (res, rej) => {
+    let fps = await getVideoFileFPS(options.inputPath)
+    // On small/medium resolutions, limit FPS
+    if (options.resolution !== undefined &&
+        options.resolution < VIDEO_TRANSCODING_FPS.KEEP_ORIGIN_FPS_RESOLUTION_MIN &&
+        fps > VIDEO_TRANSCODING_FPS.AVERAGE) {
+      fps = VIDEO_TRANSCODING_FPS.AVERAGE
+    }
+
     let command = ffmpeg(options.inputPath, { niceness: FFMPEG_NICE.TRANSCODING })
                     .output(options.outputPath)
-                    .preset(standard)
+    presetH264(command, options.resolution, fps)
```

Comment:
you need to assign the returned 'command' instance. Also `presetH264` is now `async function`, so you need to prefix its call with `await`.

Reply:
Done. Although the return value isn't really needed afaik, because it's still the same object.